packages/http-signature-utils/src/utils/signatures.ts

```diff
@@ -36,15 +28,19 @@ export const createSignatureHeaders = async ({
   if (request.body) {
     components.push('content-digest', 'content-length', 'content-type')
   }
-  const { headers } = await httpsig.sign(request, {
-    components,
-    parameters: {
-      created: Math.floor(Date.now() / 1000)
+
+  const signingKey = createSigner(privateKey, 'ed25519', keyId)
+
+  const { headers } = await httpbis.signMessage(
+    {
+      key: signingKey,
+      name: 'sig1',
+      params: ['alg', 'keyid', 'created'],
```

Comment:
We need to explicitly specify the params for the signature.

Reply:
In the future, I could see allowing for an arg to `createSignatureHeaders` that will take in a list of `params` that will be passed down into `signMessage`